config/grafana/dashboard.json

```diff
@@ -0,0 +1,1069 @@
+{
```

Comment:
This is the same dashboard as **idsvr-dashboard.json** in the root?

Reply:
It's almost the same. The only difference is that one of them is parameterized and the other isn't. Grafana does not support to provision parameterized dashboards: https://github.com/grafana/grafana/issues/10786

Reply:
I couldn't find a way to get it to work with one and the same dashboard :( Any suggestions are appreciated.
I decided to keep the parameterized dashboard because it will be the one used for importing whereas the other one is only used for "Quickstart".